src/index.js

```diff
@@ -204,25 +213,31 @@ export default {
     command.push('--auto-correct')
     command.push(filePath)
 
-    const output = await executeRubocop(getBaseExecutionOpts(filePath), command)
-    const {
-      files,
-      summary: { offense_count: offenseCount },
-    } = parseFromStd(output.stdout, output.stderr)
-
-    const offenses = files && files[0] && files[0].offenses
-
-    if (offenseCount === 0) {
-      atom.notifications.addInfo('Linter-Rubocop: No fixes were made')
-    } else {
-      const corrections = Object.values(offenses)
-        .reduce((off, { corrected }) => off + corrected, 0)
-      const message = `Linter-Rubocop: Fixed ${pluralize('offenses', corrections, true)} of ${offenseCount}`
-      if (corrections < offenseCount) {
-        atom.notifications.addInfo(message)
+    let output
+    try {
+      output = await executeRubocop(getBaseExecutionOpts(filePath), command)
```

Comment:
I think this is incorrect because all errors will be shown onto the linter pane.